src/App.scss

```diff
@@ -1,6 +1,7 @@
 // import hebbo:
 @import 'https://fonts.googleapis.com/css?family=Heebo:400,500,700&display=swap';
 @import './resources/variables.scss';
+@import './shared/shared.css';
```

Comment:
The widget component could import its own style

Reply:
you right, i also think we better have a Widget react component at our project 
and i think it could be addressed at next PR